src/twisted/test/proto_helpers.py

```diff
@@ -122,6 +122,9 @@ class StringTransport:
     @ivar disconnecting: A C{bool} which is C{False} until L{loseConnection} is
         called, then C{True}.
 
+    @ivar aborted: A L{bool} which is C{False} until L{abortConnection} is
```

Comment:
The interface what is coming from, aborted? I don't think it's on any interface: I *ad hoc*'d it up.